teuthology/nuke.py

```diff
@@ -442,16 +442,16 @@ def stale_openstack_instances(ctx, instances, locked_nodes):
 
 
 def openstack_delete_volume(id):
-    sh("openstack volume delete " + id + " || true")
+    OpenStack().run("volume delete " + id + " || true")
```

Comment:
My bad: in an earlier version I had run as a static function but it's no longer the case.